shared/utils/emoji.ts

```diff
@@ -214,6 +220,23 @@ export const search = ({
   });
 };
 
+export const getCustomEmojis = async (
```

Comment:
This should be in the `EmojiStore` like other data fetching, and store the results in the store